### DISCUSSION TITLE
Having Issues getting container running

### DISCUSSION THREAD

**Discussion by Betamax21** [qdm12/gluetun#2128] [Q&A]
Hello, hopefully, someone here can give me some advice as to what in the world is going on here. I haven't changed anything in the YAML file, but after running a Pull command I am getting endless errors over and over related to

I've included my logs here for diagnostic purposes.
[rutorrent_gluetun-2024-02-24T22-07-28.log](https://github.com/qdm12/gluetun/files/14394570/rutorrent_gluetun-2024-02-24T22-07-28.log)

**Comment by ezekieldas** [comment]
The log you provided looks like some minor tweaking would get this working. Are you still having trouble.
